S153 jira server docs (#439)
Worklytics/psoxy: docs/sources/atlassian/jira-server.md

NOTE: derived from [worklytics-connector-specs](../../infra/modules/worklytics-connector-specs/main.tf); refer to that for definitive information.

1. Follow the instructions to create a [Personal Access Token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) in your instance.
   As this is coupled to a specific User in Jira, we recommend first creating a dedicated Jira user
   to be a "Service Account" in effect for the connection (name it `svc-worklytics` or something).
   This will give you better visibility into activity of the data connector as well as avoid
   connection inadvertently breaking if the Jira user who owns the token is disabled or deleted.

>    That service account will require to have *READ* permissions over Jira, to cover following "Issues" and their worklogs and comments,

Reviewer: ```suggestion
   That service account must have *READ* permissions over your Jira instance, to be able to read issues, worklogs and comments,
```

   including their changelog where possible.

>    In terms of classical scopes if there is a need to specify someone:

Reviewer: ```suggestion
   If you're required to specify a classical scope, you can add:
```

     - `read:jira-work`

> 2. Disable or mark a proper expiration of the token.

Reviewer: ```suggestion
2. Disable or set a reasonable expiration time for the token. If you set an expiration time, it is your responsibility to re-generate the token and reset it in your host environment to maintain your connection
```

3. Copy the value of the token in `PSOXY_JIRA_SERVER_ACCESS_TOKEN` variable as part of AWS System

>    Manager parameters store / GCP Cloud Secrets (if default implementation)

Reviewer: ```suggestion
   Manager Parameter Store / GCP Cloud Secrets 
```

   NOTE: If your token has been created with expiration date, please remember to update it before

>    that date to ensure connector is going to work.

Reviewer: remove this; better inline with step (2) above.
